.github/workflows/preview.yml

```diff
@@ -0,0 +1,23 @@
+name: Build & Deploy preview
+on:
+  push:
+    branches:
+      - main
```

Comment:
you can also add paths, to trigger it only when icons were changed
    paths:
      - 'packages/icons/**/*'